### PR TITLE
Fix some style issues in engine code

### PR DESCRIPTION
**Pull request by iSazonov**
## PR Summary

The PR resolves some style issues in engine code. (Moved from PR #7242. )

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [x] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

---
This change is [Reviewable](https://reviewable.io/reviews/powershell/powershell/7246)

### CODE REVIEW

File: src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs [hunk 1]
```diff
@@ -3268,14 +3286,15 @@ internal bool LoadModuleManifestData(ExternalScriptInfo scriptInfo, ManifestProc
...
 
                             // Update the exports to only contain things that are also in the manifest export list
                             List<PSVariable> updated = new List<PSVariable>();
                             foreach (PSVariable element in ss.Internal.ExportedVariables)
                             {
-                                if (SessionStateUtilities.MatchesAnyWildcardPattern(element.Name, exportedVariables,
-                                    false))
+                                if (SessionStateUtilities.MatchesAnyWildcardPattern(element.Name,
+                                                                                    exportedVariables,
+                                                                                    defaultValue: false))
```
Comment (BrucePay) [inline src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs]: I would suggest lining up all three parameters but with less indentation.
```
           if (SessionStateUtilities.MatchesAnyWildcardPattern(
                   element.Name,
                   exportedVariables,
                   defaultValue: false))
```

File: src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs [hunk 2]
```diff
@@ -4329,15 +4349,15 @@ internal static ModuleLoggingGroupPolicyStatus GetModuleLoggingInformation(out I
...
-        /// <param name="moduleManifestPath">The manifest that generated the hashtable</param>
-        /// <param name="key">the table key to use</param>
-        /// <param name="manifestProcessingFlags">Specifies how to treat errors and whether to load elements</param>
-        /// <param name="result">Value from the manifest converted to the right type</param>
-        /// <returns><c>true</c> if success; <c>false</c> if there were errors</returns>
+        /// <param name="data">The hashtable to look for the key in.</param>
+        /// <param name="moduleManifestPath">The manifest that generated the hashtable.</param>
+        /// <param name="key">the table key to use.</param>
+        /// <param name="manifestProcessingFlags">Specifies how to treat errors and whether to load elements.am>
+        /// <param name="result">Value from the manifest converted to the right type.</param>
```
Comment (BrucePay) [inline src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs]: What's the "am" here?

File: src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs [hunk 3]
```diff
@@ -4624,19 +4644,19 @@ internal static Collection<string> GetResolvedPathCollection(string filePath, Ex
...
         }
 
         /// <summary>
-        /// Removes a module from the session state
+        /// Removes a module from the session state.
         /// </summary>
-        /// <param name="module">module to remove</param>
-        ///  <param name="moduleNameInRemoveModuleCmdlet">module name specified in the cmdlet</param>
+        /// <param name="module">module to remove.</param>
+        ///  <param name="moduleNameInRemoveModuleCmdlet">module name specified in the cmdlet.</param>
```
Comment (BrucePay) [inline src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs]: Extra space before "<param" ?

File: src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs [hunk 4]
```diff
@@ -1053,9 +1059,14 @@ private static void SortAndRemoveDuplicates<T>(List<T> input, Func<T, string> ke
         /// <param name="variablePatterns">Patterns describing the variables to export</param>
         /// <param name="doNotExportCmdlets">List of Cmdlets that will not be exported,
         ///     even if they match in cmdletPatterns.</param>
```
Comment (BrucePay) [inline src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs]: Extra spaces before 'even'

**Review by BrucePay — DISMISSED** [review]
There are a couple of things I think you should fix.

**Comment by iSazonov** [comment]
@BrucePay Your feedback was addressed.

File: src/System.Management.Automation/engine/InitialSessionState.cs [hunk 5]
```diff
@@ -3643,7 +3642,7 @@ private static void ThrowTypeOrFormatErrors(string resourceString, string errorM
         }
 
         /// <summary>
-        /// Need to have SnapIn support till we move to modules
+        /// Need to have SnapIn support till we move to modules.
```
[the remark this answers is not in the document]
Comment (iSazonov) [reply src/System.Management.Automation/engine/InitialSessionState.cs]: The PR is only style issues - please open new tracking Issue.

**Comment by iSazonov** [comment]
@BrucePay Please update your review.

**Comment by TravisEz13** [comment]
@BrucePay  Can you update your review?

**Comment by iSazonov** [comment]
@TravisEz13 @BrucePay @daxian-dbw Could you please review?

File: src/System.Management.Automation/engine/InitialSessionState.cs [hunk 6]
```diff
@@ -1042,9 +1042,9 @@ public int Count
             }
         }
 
-        //To find the entries based on name.
-        //Why collection - Different SnapIn/modules and same entity names.
-        //If used on command collection entry, then for the same name, one can have multiple output
+        // To find the entries based on name.
+        // Why collection - Different SnapIn/modules and same entity names.
+        // If used on command collection entry, then for the same name, one can have multiple output
```
Comment (daxian-dbw) [inline src/System.Management.Automation/engine/InitialSessionState.cs]: These comments are duplicate with the ones below in <summary>. They can be removed.

File: src/System.Management.Automation/engine/InitialSessionState.cs [hunk 7]
```diff
@@ -1716,10 +1716,10 @@ public static InitialSessionState Create(string[] snapInNameCollection, out PSCo
             return new InitialSessionState();
         }
 
-        //This one is for module. Can take only one module, and not collection
-        //public static InitialSessionState Create(Module);
+        // This one is for module. Can take only one module, and not collection
+        // public static InitialSessionState Create(Module);
```
Comment (daxian-dbw) [inline src/System.Management.Automation/engine/InitialSessionState.cs]: Seems to be these two can be removed.

File: src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs [hunk 8]
```diff
@@ -4457,15 +4478,15 @@ internal static ModuleLoggingGroupPolicyStatus GetModuleLoggingInformation(out I
...
-        /// <param name="data">The hashtable to look for the key in</param>
-        /// <param name="moduleManifestPath">The manifest that generated the hashtable</param>
-        /// <param name="key">the table key to use</param>
-        /// <param name="manifestProcessingFlags">Specifies how to treat errors and whether to load elements</param>
-        /// <param name="result">Value from the manifest converted to the right type</param>
-        /// <returns><c>true</c> if success; <c>false</c> if there were errors</returns>
+        /// <param name="data">The hashtable to look for the key in.</param>
+        /// <param name="moduleManifestPath">The manifest that generated the hashtable.</param>
+        /// <param name="key">the table key to use.</param>
+        /// <param name="manifestProcessingFlags">Specifies how to treat errors and whether to load elements.>
```
Comment (daxian-dbw) [inline src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs]: `</param>` is missing

**Review by daxian-dbw — COMMENTED** [review]
Left a few comments

**Review by daxian-dbw — APPROVED** [review]
@iSazonov Can you please update the PR title and summary to be more related to the changes?

**Comment by iSazonov** [comment]
@daxian-dbw Title and summary was updated.

**Review by TravisEz13 — APPROVED** [review]
> Reviewed 2 of 12 files at r1, 9 of 11 files at r3, 7 of 11 files at r4, 2 of 2 files at r5.
> *[Reviewable](https://reviewable.io/reviews/powershell/powershell/7246)* status: :shipit: complete! all files reviewed

**Pull request merged**